infra/ballista/build/lambda-handler.py

```diff
@@ -0,0 +1,190 @@
+import base64
+import os
+import subprocess
+from subprocess import PIPE
+import logging
+import pexpect
+from pexpect import popen_spawn
+import time
+import sys
+
+
+logging.getLogger().setLevel(logging.INFO)
+
+# Create global variables for the forked processes
+process_s = None
+process_e = None
```

Comment:
I think these two don't need to be global

Reply:
they do now with the rafactoring. In order to check if the processes are ok